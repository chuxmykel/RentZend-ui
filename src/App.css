*,
*::before,
*::after {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

.App {
  height: 100vh;
}

.header {
  display: flex;
}

.header a {
  text-decoration: none;
  display: inline-block;
  padding: 5px 10px 5px 5px;
  color: white;
  transition: all .2s;
  font-size: 16px;
}

.header a:hover {
  transform: scale(1.05);
}

.header a:active {
  transform: scale(.9);
}