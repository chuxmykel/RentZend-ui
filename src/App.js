import React from 'react';
import ApolloClient from 'apollo-boost';
import { ApolloProvider } from '@apollo/react-hooks';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import AgentsApplication from './containers/AgentsApplication';
import AgentsView from './containers/AgentsView';
import Header from './containers/Header';
import './App.css';

const client = new ApolloClient({
  uri: 'http://localhost:8000/graphql',
});

function App() {
  return (
    <Router>
      <ApolloProvider client={client}>
        <div className="App">
          <Header />
          <Switch>
            <Route exact path="/" component={AgentsView} />
            <Route exact path="/apply" component={AgentsApplication} />
          </Switch>
        </div>
      </ApolloProvider>
    </Router>
  );
}

export default App;
