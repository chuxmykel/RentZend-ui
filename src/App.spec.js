import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('<App /> with no props', () => {
  const container = shallow(<App />);
  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  });

  it('should have the main app div', () => {
    expect(container.find('div').length).toEqual(1);
  });

  it('should have proper props for email field', () => {
    expect(container.find('div').props()).toEqual({
      className: 'App',
      children: 'Hello World'
    });
  });
});
