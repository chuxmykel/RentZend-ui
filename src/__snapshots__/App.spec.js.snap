// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> with no props should match the snapshot 1`] = `"<div class=\\"App\\">Hello World</div>"`;
